src/modules/core/components/MemberReputation/MemberReputation.tsx

```diff
@@ -60,6 +64,10 @@ const MemberReputation = ({
     userReputationData?.userReputation,
     totalReputation?.userReputation,
   );
+  const formattedReputationPoints = getFormattedTokenValue(
+    new Decimal(userReputationData?.userReputation || 0).toString(),
+    18,
```

Comment:
Reputation should have the decimals of the native token, as far as I remember.

Maybe check with Alex, but I think this should change

Reply:
You are right! Forgot to change this